### DISCUSSION TITLE
The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'witcheroriginaddon'!

### DISCUSSION THREAD

**Discussion by CelinToKaszan** [FabricMC/fabric-api#3569] [Player support]
### Description

Error code:-1 

Honestly i have no clue what's going on.

I'm a beginner here, and I found a similar post but with a different crash report, also I didn't understand anything, so if you could explain it to me, it would mean a world to me.

Thanks in advance

[crash-2024-02-04_05.52.41-client.txt](https://github.com/FabricMC/fabric/files/14155557/crash-2024-02-04_05.52.41-client.txt)

**Comment by Daomephsta** [comment]
`witcheroriginaddon` is not compatible with Minecraft 1.19.3. It appears to have been built for 1.19.0.
x.y.z to x.y.z+1 version changes used to tend not to break mods, but then Mojang changed their versioning system and started making bigger changes in those versions.   
`witcheroriginaddon` declares itself to be compatible with 1.19.0 or later, so Fabric thinks it's compatible with 1.19.3 and attempts to load it. However it then crashes when it attempts to use Fabric features that had to change in breaking ways due to breaking changes in Minecraft.

You can't fix this yourself, only report the issue to the author.
